src/view/scripts.inc

```diff
@@ -24,6 +24,14 @@ window.SKOSMOS = {
   {%- if request.plugins.callbacks ~%}
   "pluginCallbacks": [{% for function in request.plugins.callbacks %}{% if not loop.first %}, {% endif %}"{{ function }}"{% endfor %}],
   {%- endif ~%}
+  {%- if custom_labels -%}
+  "customLabels": { 
+    {%- for k, v in custom_labels -%}
+      {%- set label = v['label'][request.lang] -%}
+      {%- set description = v['description'][request.lang] -%}
+      {%- if not loop.first -%}, {%- endif -%}"{{k}}": ["{{label[:1]|upper ~ label[1:]}}", "{{description[:1]|upper ~ description[1:]}}"]{%- endfor -%} 
```

Comment:
I wonder if there's a danger here that the custom labels may contain special characters like `"` or `\` which could break the JS syntax?

OTOH the same problem also applies to other values generated above, for example concept labels under `"prefLabels"`. They too could in principle contain special characters.

It's an unlikely scenario though...

The way to avoid it would be to generate the data structure on the PHP side and return a JSON encoded string that could be inserted here using a single Twig expression.